app/controllers/application_controller.rb

```diff
@@ -1,5 +1,9 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    @current_user = User.first
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  def configure_permitted_parameters
+    attributes = %i[name bio]
+    devise_parameter_sanitizer.permit(:sign_up, keys: attributes)
+    devise_parameter_sanitizer.permit(:account_update, keys: attributes)
   end
 end
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** => **`app/controllers/posts_controller.rb:`** fle: Kindly make sure the user is  still able to create a **Post** as right now when trying to create a post it show the below error:
![Screenshot from 2022-09-01 01-16-56](https://user-images.githubusercontent.com/83042742/187802705-b166ed60-00b5-4e82-b3c8-ad8c10899311.png)